gamemodes/husklesph/gamemode/sv_rounds.lua

```diff
@@ -1,3 +1,5 @@
+include("sv_awards.lua")
```

Comment:
I'm not sure exactly what the standard is here, but I don't think other files do their own includes and just depend on the ordering in the init/shared file. Thoughts?

Reply:
I'm not opposed to just having this in init, but it also doesn't feel necessary since (at the moment at least) nothing else relies on the contents of sv_awards.